docs/2/technical/microshare-platform-advanced/bundler-configuration.md

```diff
@@ -0,0 +1,579 @@
+---
+layout: docs
+title: Incident Bundler Configuration
+description: Let's take a look at how to configure an alert bundler for incidents
+toc: true
+---
+
+{% include image.html url="/assets/img/banner-2.jpg" description="thumbnail" %}
+
+<br>
+
+---------------------------------------
+
+##### SUMMARY: 
+
+1. [Microshare Incidents](./#1-microshare-incidents)
+2. [Overview](./#2-overview)
+3. [Default Configuration](./#3-default-configuration)
+    - A. [Configure by event type](./#a-configure-by-event-type)
+    - B. [Configure by Location](./#b-configure-by-location)
+4. [Nested Configurations](./#4-nested-configurations)
+5. [Configuring Bundler for Custom event types and locations](./#5-configuring-bundler-for-custom-event-types-and-locations)
+
+---------------------------------------
+
+## 1. Microshare Incidents
+---------------------------------------
+In the Microshare data pipeline, we process our IoT data (from sensors) into events and alerts, leading to a more responsive system where motion and environmental data are transformed into alerts and action data.
+
+An alert generates an incident that is reported to a customer as a notification. Furthermore, multiple alerts, corresponding to the same incident are bundled together by a bundler robot.
+
+This page describes how to configure a bundler robot which will bundle certain types of alerts for a given incident.
+
+## 2. Overview
+---------------------------------------
+An incident bundler robot collects multiple alerts from sensors and consolidates related alerts into a single incident. This process involves analyzing incoming alerts, identifying correlations, and grouping them based on predefined criteria which are given by the bundler robot configuration . Once the alerts are consolidated into a single incident, a notification is triggered and sent to the respective customer.
+
+The bundler robot relies on a JSON configuration to understand how to consolidate alerts into an incident. This configuration file contains the rules and parameters that dictate how alerts should be grouped. By using this configuration format, the bundler robot can adapt to various scenarios and requirements, providing accurate and efficient incident management.
+
+## 3. Default Configuration
+---------------------------------------
+The default configuration is the config that is applied to every alert encountered by the bundler. All incoming alerts are bundled into incidents by the bundler using the default configuration. 
+
+Example Default config
+
+```
+"config": {
+    "version" : "2.0.0"
+}
+```
+
```

Comment:
Can we have the full robot here and explain that's the robot and the config part look like that. 
![Screenshot 2024-08-27 at 12 33 39](https://github.com/user-attachments/assets/f4c1ebc0-26d5-40cd-9c46-13acfc9369c2)


function main(text, auth) {
    print("bundler")
    try{
        var bundler = require('./libs/products/bundler');
        var config = {
            "version" : "2.0.0"
        }
        bindings.auth = auth
        bundler.action(text, config)
    } catch (error) {
        print(error)
    }
        
    
}

Reply:
we should also explained that the recType of the robot matches the recType of the alerts we are working on. This should also be exxplained and link to the other ticket. We can allow our user to have for their custom events 
io.microshare.event.alert.custom so they can build their own config on it if they want.